src/System.Diagnostics.DiagnosticSource/src/MatchingRefApiCompatBaseline.netstandard1.3.txt

```diff
@@ -0,0 +1,5 @@
+Compat issues with assembly System.Diagnostics.DiagnosticSource:
+TypesMustExist : Type 'System.Diagnostics.Activity' does not exist in the implementation but it does exist in the contract.
```

Comment:
@lmolkova are these API intentionally not in the netstandard1.3 ref assembly while they are in the implementation?

Reply:
nope, it is not Activity should be available in netstandard 1.3, both in ref and in the implementation

Reply:
@lmolkova thanks for the confirmation. I fixed up the ref project configurations https://github.com/dotnet/corefx/pull/27881/commits/6a241bd7f9464a9c6a519095238dfa4b684bc2d8